### PR TITLE
Getting Further: Automation part 1

### PR DESCRIPTION
**Pull request by psiinon**
This look like the right level?
Anything missing at this stage?
Anyone disagree with the option scoring?
All feedback appreciated!

### CODE REVIEW

File: site/content/docs/_index.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ groups:
         link: /getting-started/
         desc: a good place to start if you are new to ZAP
     
+      - name: 'Getting Further Guides'
+        link: /docs/getting-further/
+        desc: in depth guides covering different ways you can use ZAP
+    
       - name: 'Desktop User Guide'
         link: /docs/desktop/
         desc: the help included with the ZAP desktop application
```
Comment (kingthorin) [inline site/content/docs/_index.md:18]: I'd suggest swapping this two. 

1. Started.
2. User guide.
3. Further.
4. Dev guide.

File: site/content/docs/getting-further/_index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+layout: links
+links:
+  - name: 'Getting Further With Automation'
+    link: /docs/getting-further/automation/
+    desc: an in-depth guide to ZAP automation
+
+cascade:
+  EditableContent: true
+---
+More advanced guides - [let us know](https://groups.google.com/group/zaproxy-users) what else you would like them to cover:
```
Comment (kingthorin) [inline site/content/docs/getting-further/_index.md:13]: Period not colon?

File: site/content/docs/getting-further/automation/automation-options.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+title: "Automation Guide - Options"
+tags: 
+- automation
+- guide
+type: page
+---
+
+If you want to use ZAP for automated security scanning then you have a wide range of options, also listed on the main 
+[Automate](http://localhost:3000/docs/automate/) page.
```
Comment (kingthorin) [inline site/content/docs/getting-further/automation/automation-options.md:10]: That's not gonna work  :grinning:

File: site/content/docs/getting-further/automation/automation-options.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+These automation options also have different focuses:
+
+### ZAPit
+
+[ZAPit](/blog/2023-10-18-zapit/) provides a quick ‘reconnaissance’ scan of the URL specified via the command line. 
+It is safe to run against production services but is very limited in its scope as it currently only checks the 
+URL you specify and does not explore the app any further.
+
+### Quick Start command line
```
Comment (kingthorin) [inline site/content/docs/getting-further/automation/automation-options.md:33]: > Quick Start Command Line

File: site/content/docs/getting-further/automation/automation-options.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+The [Quick Start command line](/docs/desktop/addons/quick-start/cmdline/) option is easy to run, but with very limited options so only suitable for simple scans.
```
Comment (kingthorin) [inline site/content/docs/getting-further/automation/automation-options.md:35]: Here too.

File: site/content/docs/getting-further/automation/automation-options.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+ZAP will run the standard [Spider]() to explore the site and then run the Active Scanner with the default scan policy.
```
Comment (kingthorin) [inline site/content/docs/getting-further/automation/automation-options.md:36]: Linkify.

File: site/content/docs/getting-further/automation/automation-options.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+### Docker Packaged Scans
+The [Docker Packaged Scans](/docs/docker/) are the easiest way to get started with ZAP automation and provide a reasonable amount of flexibility.
+They allow you to configure the most commonly used option via the command line and allow you to make ZAP API calls at each stage of the scan via [Scan Hooks](/docs/docker/scan-hooks/).
```
Comment (kingthorin) [inline site/content/docs/getting-further/automation/automation-options.md]: option ➡ options
(plural)

File: site/content/docs/getting-further/automation/automation-options.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+If you need to get ZAP to handle authentication for your app then the Automation Framework is likely to be a better option for you.
+The Packaged Scans only run in Docker containers and are being migrated to use the Automation Framework under the covers.
+
+### GitHub Actions
+The [GitHub Actions](https://github.com/marketplace?query=zap&verification=verified_creator) are the Packaged Scans 
+made available on the GitHub Marketplace.
+They also provide the option to raise issues directly in GitHub for any potential vulnerabilities found.
```
Comment (kingthorin) [inline site/content/docs/getting-further/automation/automation-options.md:48]: > on GitHub

File: site/content/docs/getting-further/automation/automation-options.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+### Automation Framework
+The [Automation Framework](/docs/automate/automation-framework/) is a very flexible option not tied to any container technology.
+It allows you to define sequential jobs which perform specific ZAP actions such as Spidering and Active Scanning
+and is recommended for most non-trivial automation.
+
+### API and Daemon mode
```
Comment (kingthorin) [inline site/content/docs/getting-further/automation/automation-options.md:55]: > API and Daemon Mode

File: site/content/docs/getting-further/automation/automation-options.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+The [API and Daemon mode](/docs/api/) give you full control over ZAP via a comprehensive API.
```
Comment (kingthorin) [inline site/content/docs/getting-further/automation/automation-options.md:56]: Here too.

File: site/content/docs/getting-further/automation/automation-options.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+It is very powerful but you do need to have a much better understanding of how ZAP works.
+As a result it is only recommended for system integrators or people who have need deep integration with ZAP.
```
Comment (kingthorin) [inline site/content/docs/getting-further/automation/automation-options.md:58]: > need for deep

File: site/content/docs/getting-further/automation/exploring-your-app.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
+---
+title: "Automation Guide - Exploring Your App"
+tags: 
+- automation
+- guide
+type: page
+---
+
+ZAP cannot attack parts of the target app if it doesn't not know about them.
```
Comment (kingthorin) [inline site/content/docs/getting-further/automation/exploring-your-app.md:9]:
```suggestion
ZAP cannot attack parts of the target app if it doesn't know about them.
```

**Comment by psiinon** [comment]
Thanks - think I've fixed those..

File: site/content/docs/_index.md (hunk 1 above)
Comment (psiinon) [reply site/content/docs/_index.md:18]: I think the current order is better - the User Guide is not easy to read, its more of a reference guide we link to..

Comment (kingthorin) [reply site/content/docs/_index.md:18]: Okay.

File: site/content/docs/getting-further/automation/automation-options.md (hunk 8 above)
Comment (kingthorin) [reply site/content/docs/getting-further/automation/automation-options.md:48]: Missed

Comment (psiinon) [reply site/content/docs/getting-further/automation/automation-options.md]: Ah! I had changed "on the GitHub Marketplace" to "on GitHub.." 😉

Comment (kingthorin) [reply site/content/docs/getting-further/automation/automation-options.md]: Oops, yup!

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**